identity/validator_set.go

```diff
@@ -504,11 +524,12 @@ func (vs *ValidatorStore) GetEndBlockUpdate(ctx *ValidatorContext, req types.Req
 				logger.Fatalf("failed  set last purge height, validator: %s", addrHuman)
 			}
 		}
-
 		// delegation
 		ctx.Delegators.UpdateWithdrawReward(height)
+		// vote requests check
+		vs.ExecuteAllegationTracker(ctx, activeCount)
```

Comment:
Just a question. Are we using a dynamic active validator list count to calculate the minimum votes required to pass an allegation?